source/clear-linux/tutorials/kata_migration.rst

```diff
@@ -0,0 +1,86 @@
+.. _kata_migration:
+
+Clear Containers to Kata Containers\* migration
+###############################################
+
+`Clear Containers`_ and `Kata Containers`_ can co-exist in the same system; at the moment both can be installed through the :file:`containers-virt bundle`. However, we recommend that you migrate to Kata Containers because Clear Containers is no longer maintained_ and will soon be deprecated on |CL|.
+
+.. _Clear Containers: https://github.com/clearcontainers
+.. _Kata Containers: https://github.com/kata-containers
+.. _maintained: https://github.com/kata-containers/documentation/blob/master/Upgrading.md#maintenance-warning
+
+To switch to Kata Containers in |CL|, complete the following:
+
+Stop Clear Containers instances
+*******************************
+
+Before making any changes, all running Clear Containers must be stopped:
+
+.. code-block:: bash
+
+    for container in $(sudo docker ps -q); do sudo docker stop $container; done
+
+.. note::
+
+    This step assumes that Clear Containers is on a Docker\* system.
```

Comment:
Could you please move this note to appear after line 13 above? 
I believe this assumption should be stated up front.